### ISSUE TITLE
Dictionary in Python

### ISSUE THREAD

**Issue by akash435** [prathimacode-hub/Awesome_Python_Scripts#190]
---
Title: Dictionary 
About: a dictionary created using tkinter GUI and json data file
Name: Akash Ramanand Rajak
Label: feature request
Assignee: ''

---

Define You:

- [x] LGM-SOC'21 Participant 
- [ ] Contributor

I will create a python script to create a E-dictionary. 
In this dictionary also developed the cases like
   i.) printing multiple meaning for word having multiple meanings
   ii.) printing the meaning of closest word in case of spelling mistake or typo
   iii.) also deals when the word is noun or title

**Comment by akash435** [comment]
@prathimacode-hub , please assign me this under LGM SOC, want to add this dictionary in GUIScripts.

**Comment by prathimacode-hub** [comment]
That sounds interesting. But a dictionary project is currently under review. Once PR is approved. You can add it as an enhancement. Else you can take up a new issue to work on. @akash435

**Comment by akash435** [comment]
ok

**Comment by akash435** [comment]
@prathimacode-hub , can i start working on this as an enhancement, like 
     i will improve the gui in better way,
     with searh button , also will add clear button to clear text,
     if possible will add auto complete feature
If yes, please assign this to me under LGM SOC

**Comment by prathimacode-hub** [comment]
Are you sure? First check this project and let me know. https://github.com/prathimacode-hub/Awesome_Python_Scripts/tree/main/GUIScripts/English%20Dictionary

**Comment by akash435** [comment]
I am sure about, below two things
 - i will improve the gui in better way,
 - with searh button , also will add clear button to clear text,
and will try to add autocomplete feature

**Comment by prathimacode-hub** [comment]
Ok, great. This issue is assigned to you as an enhancement. But do create a seperate README file for your enhancement. Don't do changes in the existing README which is merged. Other than that, it's all open to enhancement. 👍@akash435

**Comment by akash435** [comment]
ok
